code/datums/craft/recipes/robot.dm

```diff
@@ -188,8 +188,8 @@
 		list(/obj/item/rig, 1, "time" = 20), // Any rig as base
 		list(/obj/item/clothing, 1, "time" = 30),
 		list(/obj/item/storage/belt, 1, "time" = 30),
-		list(/obj/item/clothing/suit/armor/vest/soteriasuit, 1, "time" = 30),
-		list(/obj/item/clothing/head/helmet/soteriasuit, 1, "time" = 30),
+		list(/obj/item/clothing/suit/armor/vest/technomancersuit, 1, "time" = 30),//Now cooperate with guild young scientist
```

Comment:
You already supposed to do that via gathering some tool_upgrades from them, and with LSS since you don't have ameridian core on your own(unless it was fixed and Ameridian lab now have a reason to exist)